fuse/dl/lightning/pl_funcs.py

```diff
@@ -36,6 +36,48 @@
 from fuse.eval import MetricBase
 from fuse.eval.metrics.utils import PerSampleData
 
+# for clearml
+from clearml import Task
```

Comment:
Add clearml to dependency list

Reply:
added to dependency in [31a8f07](https://github.com/BiomedSciAI/fuse-med-ml/pull/221/commits/31a8f07db0f58ae07ae20a38916fa994eafc684d)